packages/api/src/controllers/user.ts

```diff
@@ -560,6 +560,8 @@ app.post("/token", validatePost("user"), async (req, res) => {
     !isTest &&
     user.createdAt > EMAIL_VERIFICATION_CUTOFF_DATE
   ) {
+    // Resend the verification email
+    await sendVerificationEmail(req, user, user.stripeProductId);
```

Comment:
So we're gonna resend the verification email automatically after a signin attempt, is that it? Perhaps you want to change line 561 to check the last verification email timestamp instead of the user creation ts (not sure we have that tho)